htmd/parameterization/phi.py

```diff
@@ -108,37 +108,26 @@ def setPhi(coords, atoms, left, right, phi):
 
 def getPhi(coords, atoms):
```

Comment:
If we are leaving the `atoms` parameter free, I would just remove this function as it doesn't really return phi. It returns any dihedral angle given specific atoms. So I'd just remove the function and call it directly as `dihedralAngle(coords[atoms, :, 0])`